Crash when running docs

```
$ docker-compose run --rm docs
Creating django-docker-box_docs_run ... done
sphinx-build -b spelling -n -d _build/doctrees -D language=   . _build/spelling
Running Sphinx v4.0.2
loading translations []... not available for built-in messages
Initializing Spelling Checker 7.2.1
loading pickled environment... failed
failed: __new__() missing 1 required positional argument: 'aliased'
Ignoring wiki words
Ignoring acronyms
Ignoring Python builtins
Ignoring importable module names
Ignoring contributor names
Looking for custom word list in /tests/django/docs/spelling_wordlist
Scanning contributors

Exception occurred:
  File "/usr/local/lib/python3.8/subprocess.py", line 1702, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'git'
The full traceback has been saved in /tmp/sphinx-err-i4m5dh4k.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make: *** [Makefile:159: spelling] Error 2
ERROR: 2
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -12,6 +12,7 @@ RUN apt-get update \
           libenchant-2-2 \
           gettext \
           wget \
+          git \
     && apt-get clean
 
 RUN groupadd -r test && useradd --no-log-init -r -g test test

